src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const robotAccounter = (a) => {
+    function accounter(b) {
+      count++;
+
+      return count > 3 && !(count % 2)
```

Comment:
In this case you want to compare it explicitly to 0 make it more readable in this statement
```suggestion
      return count > 3 && count % 2 !== 0
```